evm/src/executor/inspector/cheatcodes/ext.rs

```diff
@@ -281,13 +281,29 @@ fn remove_file(state: &mut Cheatcodes, path: impl AsRef<Path>) -> Result<Bytes,
 /// it will call itself to convert each of it's value and encode the whole as a
 /// Tuple
 fn value_to_token(value: &Value) -> Result<Token, Token> {
+    println!("value to token: {:?}", value);
     if value.is_boolean() {
         Ok(Token::Bool(value.as_bool().unwrap()))
     } else if value.is_string() {
         let val = value.as_str().unwrap();
```

Comment:
these also could be `if let Some()` `as_string` and `as_boolean`, I believe these exist on `Value`